bigbluebutton-html5/imports/ui/components/actions-bar/component.jsx

```diff
@@ -30,14 +29,15 @@ export default class ActionsBar extends Component {
         <div className={styles.center}>
           <MuteAudioContainer />
           <JoinAudioOptionsContainer
-            open={this.openJoinAudio.bind(this)}
-            close={() => {exitAudio();}}
+            handleJoinAudio={this.openJoinAudio.bind(this)}
```

Comment:
You could pass down the `handleJoinAudio` from the container instead of coupling inside the ActionsBar component.